identity/credentials_oidc.go

```diff
@@ -29,6 +29,9 @@ type CredentialsOIDCProvider struct {
 	InitialIDToken      string `json:"initial_id_token"`
 	InitialAccessToken  string `json:"initial_access_token"`
 	InitialRefreshToken string `json:"initial_refresh_token"`
+	CurrentIDToken      string `json:"current_id_token"`
+	CurrentAccessToken  string `json:"current_access_token"`
+	CurrentRefreshToken string `json:"current_refresh_token"`
```

Comment:
Can you please rename these to:

```suggestion
	LastIDToken      string `json:"last_id_token"`
	LastAccessToken  string `json:"last_access_token"`
	LastRefreshToken string `json:"last_refresh_token"`
```

Reply:
ok